pkg/controller/plan/scheduler/vsphere/scheduler_test.go

```diff
@@ -83,6 +83,9 @@ func TestScheduler(t *testing.T) {
 			},
 		},
 		hostC: {
+			{
+				cost: 11,
```

Comment:
question:
in our git automation we run "make ci"
https://github.com/kubev2v/forklift/blob/main/.github/workflows/pull-request.yml#L138

and "ci" does not run "test" :-(
https://github.com/kubev2v/forklift/blob/main/Makefile#L78

do we want to run "make test", am i missing something ?

Reply:
sorry, we run "all" that run "test", resolving